crates/types/src/world_state.rs

```diff
@@ -17,8 +16,6 @@ pub struct WorldState {
     pub filtered_game_state: Option<FilteredGameState>,
     #[leaf]
     pub game_controller_state: Option<GameControllerState>,
-    #[leaf]
-    pub game_phase: GamePhase,
```

Comment:
Why has this been removed? 🤔

Reply:
Because it is duplicated information with the `GameControllerState`. If there is no `GameController`, there is no game, and thus no game phase. This was previously mocked as `GamePhase::Normal`, if there was no gamecontroller. I think this is confusing and should be handled differently. Thus, no artificial `GamePhase` wihout an actual game.